server/models/Profile.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  userName: {
+    type: String,
+    unique: true,
+    required: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  gender: {
+    type: String,
+  },
+  birthday: {
+    type: Date,
+  },
+  email: {
```

Comment:
The `email` prop is already a part of the User model, please remove it here in order to avoid duplication of data